_data/footer.yml

```diff
@@ -57,7 +57,8 @@
     social_icon: Twitter
     new_window: true
   - name: Calendar
-    link: https://bit.ly/2SbrRhe
+    link: /calendar
+    #https://bit.ly/2SbrRhe
```

Comment:
Looks great. As we go to remove the direct links to the calendar, it would be nice to include a link on the page somewhere (perhaps, under the visual calendar), so that users who desire to subscribe to the calendar may do so.